From 4a4f0979dbdcd81953b04651455e3cdbc9c1677d Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Sat, 26 Sep 2026 11:37:02 +0100
Subject: [PATCH] lineage: res: Map all apps (ALL_APPS) key

Google TV Launcher does not handle the raw keycode, but it does handle
the intent.

Change-Id: I689614cd556b523aced3c2545ce1b61558f95cca
---

diff --git a/lineage/res/res/values/arrays.xml b/lineage/res/res/values/arrays.xml
index 647d54e..0d88013 100644
--- a/lineage/res/res/values/arrays.xml
+++ b/lineage/res/res/values/arrays.xml
@@ -91,6 +91,7 @@
         <item>194</item> <!-- KEYCODE_BUTTON_7 -->
         <item>195</item> <!-- KEYCODE_BUTTON_8 -->
         <item>225</item> <!-- KEYCODE_PAIRING -->
+        <item>284</item> <!-- KEYCODE_ALL_APPS -->
     </integer-array>
 
     <!-- Array of packages for special keys on device.
@@ -105,5 +106,6 @@
         <item>com.hbo.hbonow</item>
         <item>com.android.vending</item>
         <item>com.google.android.intent.action.CONNECT_INPUT</item>
+        <item>android.intent.action.ALL_APPS</item>
     </string-array>
 </resources>
